locales-pending/premium.ftl

```diff
@@ -262,3 +262,80 @@ false-door-test-popup-close = Close
 false-door-test-phase-2-content-part-one = Did you know there are sites selling your private info?
 false-door-test-phase-2-content-part-two = Automatically remove data from sketchy sites.
 false-door-test-phase-2-content-part-two-dashboard = Automatically remove data from broker sites. 
+
+# High Risk Data Breaches
+
+high-risk-breach-heading = Here’s what to do
+high-risk-breach-subheading = This requires access to your sensitive info, so you’ll need to manually fix it.
+# Variables
+# $num_breaches is the number of breaches where the high risk data was found.
+high-risk-breach-summary = { $num_breaches ->
+  [one] It appeared in { $num_breaches } data breach:
+  *[other] It appeared in { $num_breaches } data breaches:
+}
+# Variables
+# $breach_name is the name of the breach where the high risk data was found.
+# $breach_date is the date when the breach occurred. 
+# An example of this string is Twitter on 13/09/18.
+high-risk-breach-name-and-date = { $breach_name } <breach_date>on { $breach_date }</breach_date>
+high-risk-breach-mark-as-fixed = Mark as fixed
+high-risk-breach-skip = Skip for now
+# Variables:
+# $estimated_time is the estimated time it would take for a user to complete breach resolution steps. It not be singular, and the + is meant as "or more".
+# An example of this string is Your estimated time: 15+ minutes.
+high-risk-breach-estimated-time = Your estimated time: { $estimated_time }+ minutes
+
+# Credit Card Breaches
+
+high-risk-breach-credit-card-title = Your credit card number was exposed
+high-risk-breach-credit-card-description = Anyone who gets it can make unauthorized purchases that you may be liable for. Act now to prevent financial harm.
+high-risk-breach-credit-card-step-one = If you still have this card, contact the issuer to report it stolen.
+high-risk-breach-credit-card-step-two = Request a new card with a new number.
+high-risk-breach-credit-card-step-three = Check your accounts for unauthorized charges.
+high-risk-breach-credit-card-illustration-alt = Credit card illustration
+
+# Bank Account Breaches
+
+high-risk-breach-bank-account-title = Your bank account was exposed
+high-risk-breach-bank-account-description = Taking action ASAP could give you more legal protections to help you recover any losses.
```

Comment:
Personal suggestion, probably to redirect to content: we should avoid things acronyms ASAP. In any case, it requires an explanation in a comment.